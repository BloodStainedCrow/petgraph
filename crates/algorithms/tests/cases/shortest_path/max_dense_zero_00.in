version https://git-lfs.github.com/spec/v1
oid sha256:2f8ea9e13435f6306e8fb27959ceef9895f0d5ddf7faf977b684d24fefcaf2e1
size 4836119
