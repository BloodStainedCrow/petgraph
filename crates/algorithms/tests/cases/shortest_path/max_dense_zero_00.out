version https://git-lfs.github.com/spec/v1
oid sha256:4303004b151251b7956ab2483e28bae4c0e36bbd69cca31d7c60fb0ad2c0a66d
size 12
